color-picker/src/commonMain/kotlin/com/godaddy/android/colorpicker/harmony/HarmonyColorPicker.kt

```diff
@@ -32,12 +32,44 @@ import kotlin.math.atan2
 import kotlin.math.hypot
 import kotlin.math.min
 
+@Deprecated(
+    message = "Use version with programmatic updates",
+    replaceWith = ReplaceWith(
+        """HarmonyColorPicker(
+            modifier = modifier,
+            harmonyMode = harmonyMode,
+            color = color,
+            onValueChanged = { HsvColor.from(it).let(onColorChanged) }
+        )"""
+    )
+)
 @Composable
 fun HarmonyColorPicker(
     modifier: Modifier = Modifier,
     harmonyMode: ColorHarmonyMode,
     color: Color = Color.Red,
     onColorChanged: (HsvColor) -> Unit
+) {
+    HarmonyColorPicker(
+        modifier = modifier,
+        harmonyMode = harmonyMode,
+        color = color,
+        onValueChanged = { HsvColor.from(it).let(onColorChanged) }
+    )
+}
+
+/**
+ * Color Picker that programmatically updates.
+ * Shows a brightness bar if [fixedBrightness] is null
+ * otherwise all colors are given the provided brightness value
+ */
+@Composable
+fun HarmonyColorPicker(
+    modifier: Modifier = Modifier,
+    harmonyMode: ColorHarmonyMode,
+    color: Color = Color.Red,
+    fixedBrightness: Float? = null,
+    onValueChanged: (Color) -> Unit
```

Comment:
This shouldn't be changed to a Color at this point, the API should return a HsvColor otherwise color information is lost, as multiple HsvColors map to the same Color value. This will break for users who need individual color parts of information.

Reply:
Makes sense.  I did that because I wanted to make the input and value changed output match.  So I'll create one where `color: Color` becomes `hsvColor: HsvColor` instead.